src/plugins/debugmon/debugmon.cpp

```diff
@@ -176,6 +176,8 @@ debugmon::debugmon(drakvuf_t _drakvuf, output_format_t _output)
     this->debug.data = (void*)this;
     this->debug.type = DEBUG;
     this->debug.name = nullptr;
+    this->debug.ttl = UNLIMITED_TTL;
+    this->debug.ah_cb = nullptr;
```

Comment:
Ditto

Reply:
I think this has to stay as it is. From what I understand this explicit initialization wouldn't be required if there where no user defined constructor for `debugmon`. [Godbolt](https://godbolt.org/z/zxqnf6)